onnx_caffe2/backend.py

```diff
@@ -595,18 +593,18 @@ def prepare(cls, model, device='CPU', **kwargs):
 
     @classmethod
     # TODO: This method needs a refactor for clarity
-    def _onnx_node_to_caffe2_op(cls, node_def, opset_version):
+    def _onnx_node_to_caffe2_op(cls, model_helper, node_def, opset_version):
         if node_def.op_type in cls._special_operators:
             translator = getattr(cls, cls._special_operators[node_def.op_type])
         else:
             translator = cls._common_onnx_node_to_caffe2_op
-        ops = translator(OnnxNode(node_def), opset_version)
+        ops = translator(model_helper, OnnxNode(node_def), opset_version)
         if not isinstance(ops, collections.Iterable):
             ops = [ops]
         return ops
 
     @classmethod
-    def _common_onnx_node_to_caffe2_op(cls, onnx_node, opset_version):
+    def _common_onnx_node_to_caffe2_op(cls, model_helper, onnx_node, opset_version):
```

Comment:
it's not really valid to pass model_helper here as some callers of this function expect to patch up the return results instead